tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -297,11 +300,31 @@ def get_tensorrt_rewriter_config(conversion_params, is_v2=False):
   _check_conversion_params(conversion_params, is_v2=is_v2)
 
   rewriter_config_with_trt = rewriter_config_pb2.RewriterConfig()
-  if conversion_params.rewriter_config_template is None:
+
+  if not disable_non_trt_optimizers:
```

Comment:
If template is not None do we still want to add these? How about:
```
if rewriter_config_template is None:
  ...extend(["constfold", ...])
  ...add trt optimizer
if disable_non_trt_optimizers:
  ...disable them
```

Reply:
I was wondering about the same thing.
I am not sure if `rewriter_config_with_trt.CopyFrom` respects that.

You could say since `disable_non_trt_optimizers` sounds generic, it should also affect the template.

Reply:
Yeah CopyFrom will overwrite that so we need to do that at the end of the method.

Reply:
I just moved the disablement to the end of the function. Please review.

Reply:
I merged the two if conditions.